upgrades/v3.0.5/README.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Neutron v3.0.5 Upgrade
+order: 2
+---
+
+<!-- markdown-link-check-disable -->
+
+# Neutron v3.0.5 Upgrade, Instructions
+
+- Chain upgrade point: `May 16th 2024, 15:30 UTC` approximately at height `10,280,000`;
+- Go version: `v1.21`
+- Release: https://github.com/neutron-org/neutron/releases/tag/v3.0.5
+
+This document describes the steps for validators and full node operators, to upgrade successfully to the Neutron v3.0.5 release. For more details on the release, please see the [release notes](https://github.com/neutron-org/neutron/releases/tag/v3.0.5).
+
+## Upgrade date
+
+The upgrade will take place approximately at `15:30 UTC` approximately at height ` 10280000`;
```

Comment:
I know we mentioned the date above. But, if we have a dedicated section, lets mention it one more time.